frontend/src/state/ui.ts

```diff
@@ -234,6 +234,10 @@ const defaultUiSettings = {
         quickSearch: ''
     },
 
+    rpncSecretList: {
```

Comment:
I think you meant
`rpcnSecretList` -> `rpncSecretList`

Reply:
right